Formula/fastqc.rb

```diff
@@ -0,0 +1,25 @@
+class Fastqc < Formula
+  desc "Quality control tool for high throughput sequence data"
+  homepage "https://www.bioinformatics.babraham.ac.uk/projects/fastqc/"
+  url "https://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc_v0.11.5.zip"
+  sha256 "dd7a5ad80ceed2588cf6d6ffe35e0f161c0d9977ed08355f5e4d9473282cbd66"
+
+  depends_on :java
+
+  def install
+    chmod 0755, "fastqc"
+    prefix.install Dir["*"]
+    mkdir_p bin
```

Comment:
`mkdir_p` should not be necessary, IIRC `bin.install_symlink` will create the directory